packages/openjdk-21/build.sh

```diff
@@ -2,15 +2,13 @@ TERMUX_PKG_HOMEPAGE=https://github.com/openjdk/mobile
 TERMUX_PKG_DESCRIPTION="Java development kit and runtime"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=17.0
-TERMUX_PKG_REVISION=31
-_COMMIT=82234f890786d49c49cf4ecbcb09c47bd9bea7ed
-TERMUX_PKG_SRCURL=https://github.com/openjdk/mobile/archive/$_COMMIT.tar.gz
-TERMUX_PKG_SHA256=5b298148a26e754120c6dfe699056d0609fc6ed92bfc858dc2ba4909ef6e791b
+TERMUX_PKG_VERSION=21.0
+TERMUX_PKG_SRCURL=git+https://github.com/openjdk-mobile-termux
```

Comment:
This doesn't seem valid, no org.

Reply:
Oh, maybe that was what twaik meant as well :)